metaserver/src/main/scala/scalafix/languageserver/ScalafixEnrichments.scala

```diff
@@ -7,9 +7,30 @@ import scalafix.lint.LintMessage
 import scalafix.patch.Patch
 import scalafix.rule.RuleCtx
 import scalafix.rule.RuleName
+import langserver.{types => l}
+import scala.meta.languageserver.ScalametaEnrichments._
+import scalafix.lint.LintSeverity
 
 // TODO(olafur) figure out how to refactor scalafix api so this is no longer needed.
```

Comment:
very minor, but this TODO now applies to only some of the method in `ScalafixEnrichments`

Reply:
Good catch, I've removed the comment since the internal methods are clearly document with the `Internal` suffix